WordPress/Classes/ViewRelated/System/FilterBar.swift

```diff
@@ -36,9 +42,7 @@ class FilterTabBar: UIControl {
         return divider
     }()
 
-    /// Titles of tabs to display.
-    ///
-    @IBInspectable var items: [String] = [] {
+    var tabBarItems: [FilterTabBarItem] = [] {
```

Comment:
Was initially so that they could exist side-by-side – left it in case we wanted to keep both implementations running. Happy to rename if we're just proceeding with one!

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/11172/commits/24e0daf87eaffdf5060ed1aa3d5a8bd042083bb2

Reply:
Looks like you've resolved it anyway, but I can't think of a reason we'd need to maintain both implementations :)